openreview/journal/invitation.py

```diff
@@ -2581,33 +2578,6 @@ def set_revision_submission(self, note):
                                 'markdown': True
```

Comment:
I think we should disable markdown here. There are actually 2 other locations that define the abstract field that should be changed as well.

Reply:
I fixed this in [84282c8](https://github.com/openreview/openreview-py/pull/1170/commits/84282c808a1d64ae705b00f455e1b491b2248991)